CPAC/network_centrality/network_centrality.py

```diff
@@ -144,4 +144,4 @@ def create_centrality_wf(wf_name, method_option, weight_options,
     centrality_wf.connect(sep_subbriks_node, 'output_niftis',
                           output_node, 'outfile_list')
 
-    return centrality_wf
+    return centrality_wf
```

Comment:
Don't need to change this file at all

```suggestion
    return centrality_wf

```